.storybook/pages/CoursePlannerEdit/CoursePlannerEdit.module.css

```diff
@@ -16,6 +16,22 @@
   }
 }
 
+/**
+ * Individual Menu item that displays when clicking the vertical dots.
+ */
+.menu-item {
+  padding: var(--eds-size-1-and-half) var(--eds-size-2);
+}
+
+/**
+ * Destructive variant of the Menu item.
+ *
+ * For indicating deletion of the project.
+ */
+.menu-item--destructive {
+  color: var(--eds-theme-color-text-utility-error);
+}
```

Comment:
This is something to bring up with design; menu items can have a state or variant for different cases, which we should handle with a prop. Similar to how we do `disabled`.

Since the component doesn't know about any of that now, we can leave this as-is, and follow up with a change to `Menu` AND a change to this page to use whatever new thing we introduce.

Reply:
Makes sense to me